lib/mongo/auth/stringprep/profiles/sasl.rb

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2018 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'mongo/auth/stringprep/tables'
```

Comment:
Can we remove this?

Reply:
Can we remove this?

Reply:
Sorry, missed the comment the first time (still getting used to the Github code review UI). Right now, the code doesn't work without it, but I suspect that's because in stringprep.rb the `Tables` module is loaded after the `SASL` module. I assume that it's okay to reverse the order of those?